generate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+  echo "Required input file parameter is missing."
+  exit 1
+fi
+
+input="$1"
+
+openapi-generator generate \
```

Comment:
Should this be
```suggestion
openapi-generator-cli generate \
```

From the usage example for the npm package [here](https://github.com/OpenAPITools/openapi-generator-cli#usage).